config/locales/en.yml

```diff
@@ -114,4 +114,17 @@ en:
     CONFIRM_HIDE:
       NOT_IN_USE: "Are you sure? Students will not see this question, and it will not appear in reports"
       IN_USE: "Are you sure you want to hide this element? You will lose data from %{learners} that have answered this question."
-      TRACKED: "Are you sure you want to hide this Tracked Question? You will no longer be able to see special reports for this question."
\ No newline at end of file
+      TRACKED: "Are you sure you want to hide this Tracked Question? You will no longer be able to see special reports for this question."
+  COMPLETION_PAGE:
+    HEADER: "%{activity} activity complete!"
+    CONGRATULATIONS: "Congratulations!"
+    FINISHED_ACTIVITY: "You have reached the end of \"%{activity}\" and your work has been saved."
+    UNFINISHED_ACTIVITY: "It looks like you haven't quite finished \"%{activity}\" yet. The answers you've given have been saved."
+    FINISHED_SEQUENCE: "You have completed all your work for this module!"
+    CONTINUE: "Continue on to %{next_activity}, or click \"Exit\" to close the activity."
+    EXIT: "Your work has been saved. Click \"Exit\" to close the activity."
+    START_NEXT: "Start Next Activity"
+    GENERATE_A_REPORT: "Show All Answers"
+    CONFIRM_CLOSE: "Exit %{activity}?"
+    EXIT_LABEL: "Exit"
+    NEXT_UP: "Next Up…"
```

Comment:
would be better to add a return here so future changes don't appear to be modifying the last line